generate/pydown/pydown.py

```diff
@@ -365,7 +365,13 @@ def ConstantsMd(inPythonFileName):
             # Consider symbols like Union defined in lines like the following:
             # "from typing import Union"
             tokens = line.split()
+            
             if len(tokens) > 3 and tokens[0] == 'from' and tokens[2] == 'import':
+                # from typing import Optional, Union are currently split as
```

Comment:
Oh, sorry about that. I should fix this part because I can think of cases where it still won't work. But I can do that later.